src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -24,156 +27,159 @@ case class CromwellRuntimeAttributes(docker: Option[String],
 object CromwellRuntimeAttributes {
   private val log = LoggerFactory.getLogger("RuntimeAttributes")
 
-  def apply(wdlRuntimeAttributes: RuntimeAttributes, backendType: BackendType): CromwellRuntimeAttributes = {
-    val attributeMap = AttributeMap(wdlRuntimeAttributes.attrs)
+  def apply(wdlRuntimeAttributes: RuntimeAttributes, backendCall: BackendCall): Try[CromwellRuntimeAttributes] = {
```

Comment:
While one can define them to return anything one wants I think it's odd for an `apply` not to have the return type be the actual type